tests/unit/disco/components/TestAddon.js

```diff
@@ -466,9 +466,6 @@ describe(__filename, () => {
       const reason = 'WHATEVER';
 
       const root = render({
-        addonProps: {
```

Comment:
Note: This was required to avoid an error. I could have ensured that the `current_version` used had a `files` property, but I don't think this has any impact on the test. From what I can tell it's just the `compatible: false` that controls the outcome of this test, so I believe it is fine to just remove these lines.